### DISCUSSION TITLE
Subscribing to a table doesn't load relationships to other tables

### DISCUSSION THREAD

**Discussion by kenmueller** [supabase/supabase#422] [Feature Requests]
# Feature request

## Is your feature request related to a problem? Please describe.

Subscribing to a table doesn't load relationships to other tables. This is unlike selecting elements from a table, where relationships are loaded normally.

## Describe the solution you'd like

I'd like for relationships to be loaded when a table is subscribed to, just like selecting elements with `.select(...)`.

## Describe alternatives you've considered

I've considered subscribing to the entire relationship table, but that's unfeasible. I've also considered selecting the row in other table once I get the `INSERT` event, but that's also unfeasible.

**Comment by kiwicopple** [comment]
Hey @kenmueller - you'll have to bear with us on this one as it's a surprisingly difficult problem to solve. We are working on it, but I don't have a timeline yet. I agree - it would be an amazing Dev experience when we get subscriptions like this

**Comment by kiwicopple** [comment]
@kenmueller I'm going to move this one to Discussions 👍
